andela/blackninjas-frontend
# #169817844 Users should be able to see his/her trip statistics

#### What does this PR do?
- A user on barefoot nomad should be able to see the number of trips he/she created in a given time
#### Description of Task to be completed?
- Add actions to fetch data from the api 
- add reducers to handle actions and update the state
- add responsiveness to the design across many devices
#### How should this be manually tested?
- clone this repo
- checkout to this branch and make sure you have all the packages installed by typing `npm install`
- run the server locally by typing `npm start-dev` and it should open in your favorite browser
- click on the statistics link and you can change date to see the number of trips from that date
#### Any background context you want to provide?
- this PR is not yet in production
#### What are the relevant pivotal tracker stories?
[#169817844](https://www.pivotaltracker.com/story/show/169817844)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/51836855/76774331-70a32f80-67ac-11ea-8d61-72727e6e9eae.png)
